engine/src/flutter/lib/web_ui/lib/src/engine/web_paragraph/painter.dart

```diff
@@ -81,9 +113,10 @@ class CanvasKitPainter extends Painter {
 
       paintContext.reset();
       paintContext.lineWidth = thickness;
-      paintContext.strokeStyle = block.style.decorationColor!.toCssString();
+      paintContext.strokeStyle = (block.style.decorationColor ?? const ui.Color(0x00000000))
+          .toCssString();
 
-      switch (block.style.decorationStyle!) {
+      switch (block.style.decorationStyle ?? ui.TextDecorationStyle.solid) {
```

Comment:
Now that `hasElement` is updated, we know for sure that `block.style.decorationStyle` will always have a value.
```suggestion
      switch (block.style.decorationStyle!) {
```